src/styles/blocks/header.scss

```
@@ -0,0 +1,19 @@
+$header-margin: 50px;
+
```

```suggestion
$header-margin: 50px;

```
змінна має бути в файлі з змінними

---

ця змінна локальна, більше ніде не використовується
щоб не плутатись, залишив її в рідному блоці